README.md

```diff
@@ -43,53 +43,132 @@ exports.passport = {
 ### Using github and twitter strategy
 
 ```js
-// app.js
+// config/config.default.js
+exports.passport = {
+  github: {
```

Comment:
Since this plugin is called `egg-passport`, its config should be `config.passport`. Every configuration exposed by this plugin should be in it.

If there's a another plugin, say `egg-passport-github`, config should be named after `config.passportGithub` according to plugin naming convention.